modules/apps/object/object-admin-rest-impl/src/main/java/com/liferay/object/admin/rest/internal/resource/v1_0/ObjectDefinitionResourceImpl.java

```diff
@@ -278,6 +286,37 @@ public ObjectDefinition postObjectDefinition(
 			objectDefinition.getObjectRelationships(),
 			objectDefinition.getObjectViews());
 
+		List<com.liferay.object.model.ObjectField> aggregationFields =
+			transformToList(
+				ArrayUtil.filter(
+					objectDefinition.getObjectFields(),
+					objectField -> StringUtil.equals(
+						objectField.getBusinessTypeAsString(),
+						ObjectFieldConstants.BUSINESS_TYPE_AGGREGATION)),
+				objectField -> ObjectFieldUtil.toObjectField(
+					_listTypeDefinitionLocalService, objectField,
+					_objectFieldLocalService, _objectFieldSettingLocalService,
+					_objectFilterLocalService));
+
+		if (aggregationFields != null) {
```

Comment:
@seltong I believe this will never be null. Could you confirm that? If so, you can remove this check.

Reply:
Hey @seltong! This check is already done by the for clause. You don't need to worry about it. Could you remove this if clause? Thx